apps/etterlatte-saksbehandling-ui/client/src/components/klage/stegmeny/KlageStegmeny.tsx

```diff
@@ -1,13 +1,42 @@
 import { StegMenyWrapper } from '~components/behandling/StegMeny/stegmeny'
 import React from 'react'
 import { KlageNavLenke } from '~components/klage/stegmeny/KlageNavLenke'
+import { useKlage } from '~components/klage/useKlage'
+import { Klage, KlageStatus } from '~shared/types/Klage'
+import { JaNei } from '~shared/types/ISvar'
+
+export function kanVurdereUtfall(klage: Klage | null): boolean {
+  const klageStatus = klage?.status
+
+  switch (klageStatus) {
+    case KlageStatus.UTFALL_VURDERT:
+    case KlageStatus.FORMKRAV_OPPFYLT:
+      return true
+    case KlageStatus.FERDIGSTILT:
+      return klage?.formkrav?.formkrav.erFormkraveneOppfylt === JaNei.JA
+  }
+  return false
+}
+
+export function kanSeOppsummering(klage: Klage | null): boolean {
+  const klageStatus = klage?.status
+  switch (klageStatus) {
+    case KlageStatus.FERDIGSTILT:
+    case KlageStatus.UTFALL_VURDERT:
+    case KlageStatus.FORMKRAV_IKKE_OPPFYLT:
+      return true
+  }
+  return false
+}
 
 export function KlageStegmeny() {
+  const klage = useKlage()
+
   return (
     <StegMenyWrapper>
       <KlageNavLenke path="formkrav" description="Vurder formkrav" enabled={true} />
-      <KlageNavLenke path="vurdering" description="Vurder klagen" enabled={true} />
-      <KlageNavLenke path="oppsummering" description="Oppsummering" enabled={true} />
+      <KlageNavLenke path="vurdering" description="Vurder klagen" enabled={kanVurdereUtfall(klage) || true} />
+      <KlageNavLenke path="oppsummering" description="Oppsummering" enabled={kanSeOppsummering(klage) || true} />
```

Comment:
`|| true` er her for testing enn så lenge, skal fjernes når flyten er støttet igjennom hele løpet